test/Align/test_align.py

```diff
@@ -364,25 +364,54 @@ def test_hydrogen_mass_repartitioning():
     # Create the merged molecule.
     merged = BSS.Align.merge(m0, m1, mapping, allow_ring_breaking=True)
 
+    # Create a dummy element.
+    dummy = Element("Xx")
+
+    # Get the elements in either end state.
+    elements0 = merged._sire_object.property("element0").toVector()
+    elements1 = merged._sire_object.property("element1").toVector()
+
     # Work out the initial mass of the system.
     initial_mass0 = 0
-    for mass in merged._sire_object.property("mass0").toVector():
-        initial_mass0 += mass.value()
+    for idx, mass in enumerate(merged._sire_object.property("mass0").toVector()):
```

Comment:
Could the new `sire` search functions simplify these? I.e. get the sum of the masses of the non-dummy atoms. I was under the impression that the new API can do that?

Reply:
I think so, but believe that this might need the latest 2023 updates that are yet to be merged. Here I just quickly edited the existing unit test, rather than re-writing the whole thing for the new API.